controllers/druid/handler.go

```diff
@@ -979,18 +979,18 @@ func getVolume(nodeSpec *v1alpha1.DruidNodeSpec, m *v1alpha1.Druid, nodeSpecUniq
 }
 
 func getCommand(nodeSpec *v1alpha1.DruidNodeSpec, m *v1alpha1.Druid) []string {
-  if (m.Spec.StartScript != "" && m.Spec.EntryArg != "") {
```

Comment:
This and the next set of changes look like non-semantic changes only. You might want to leave these for a different diff to ensure the policyv1 changes don't conflict with other unrelated changes but that happen to touch the same lines.